azure-quantum/azure/quantum/storage.py

```diff
@@ -104,10 +131,109 @@ def download_blob(blob_url: str) -> Any:
 
     return response
 
+def download_blob_properties(blob_url: str) -> Dict[str, str]:
+    blob_client = BlobClient.from_blob_url(blob_url)
+    logger.info(f"Downloading blob properties '{blob_client.blob_name}' from container '{blob_client.container_name}' on account: '{blob_client.account_name}'")
+
+    response = blob_client.get_blob_properties()
+    logger.debug(response)
+
+    return response
+
+def download_blob_metadata(blob_url: str) -> Dict[str, str]:
+    return download_blob_properties(blob_url).metadata
+
+def set_blob_metadata(blob_url: str, metadata: Dict[str, str]):
+    blob_client = BlobClient.from_blob_url(blob_url)
+    logger.info(f"Setting blob properties '{blob_client.blob_name}' from container '{blob_client.container_name}' on account: '{blob_client.account_name}'")
+    return blob_client.set_blob_metadata(metadata=metadata)
+
 def remove_sas_token(sas_uri: str) -> str:
     index = sas_uri.find('?')
     if index != -1:
         sas_uri = sas_uri[0:index]
     
     return sas_uri
 
+
+def init_blob_for_streaming_upload(container: ContainerClient, blob_name: str, content_type:str, content_encoding:str, data: Any, returnSasToken: bool=True) -> str:
```

Comment:
This line is a bit long, could you shorten it?

Reply:
Same comment here on camel case vs. snake case, `returnSasToken` should be `return_sas_token`